Gui/TextRenderer.cpp

```diff
@@ -336,6 +344,21 @@ TextRenderer::renderText(float x,
                 assert( glIsTexture(texture) );
             }
             glCheckError();
+#if QT_VERSION >= QT_VERSION_CHECK(5, 4, 0)
+            glBegin(GL_TRIANGLES);
```

Comment:
What is the reason for not using `GL_QUADS` here? This looks like an unnecessary change.
Quads are perfectly valid: https://www.khronos.org/registry/OpenGL-Refpages/gl2.1/xhtml/glBegin.xml

Reply:
This is more of a preference because using `GL_TRIANGLES` is more accurate than using `GL_QUADS` (see the [OpenGL wiki](https://www.khronos.org/opengl/wiki/Primitive#Quads) and this [StackOverflow thread](https://stackoverflow.com/questions/6644099/what-is-so-bad-about-gl-quads)). Could use `GL_QUADS` if that's wanted.

Reply:
GL_QUADS is OK when the quad is planar, and even more when it's a rectangle, which is the case here.
So I don't see the point in rewriting it as GL_TRIANGLES, and having a different version for Qt5

Reply:
With the deprecation of `QGLWidget::convertToGLFormat` (I think) the texture coordinates require to be flipped in the Y axis for accurate display of text through `Natron::TextRenderer`.

Reply:
Yes maybe, according to https://stackoverflow.com/a/24907855